ee/ucp/deploy-application-package.md

```diff
@@ -8,46 +8,46 @@ keywords: ucp, swarm, kubernetes, application, app package
 
 Docker Enterprise Edition 2.1 introduces application packages in Docker. With application packages, you can add metadata and settings to an existing Compose file. This gives operators more context about applications they deploy and manage.
 
-An application package can have one of these formats:
+Application packages can present in one of two different formats, Directory or Single-file:
 
```

Comment:
In my opinion, the two format types should be in bold at first mention, as well as at the head of their bullets directly below: Application packages can present in one of two different formats, **Directory** or **Single-file**: